### PR TITLE
CSS Modules support

### LINKED ISSUE #622: CSS Modules support
- vite build
- `import { Button } from '@taskany/bricks/harmony'`
- `import '@taskany/bricks/harmony/styles.css'`
- storybook themes support

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes
- [x] DX

## Related issues

Resolve #622 

## QA Instructions, Screenshots, Recordings
  
```ts
// In root file, for example, _app.tsx
import '@taskany/bricks/harmony/style.css'; // Import styles

// Default components available here
import { Button } from '@taskany/bricks'; // Import components

// Harmony components available here
import { Button } from '@taskany/bricks/harmony'; // Import components
```

`npm run build` сreates a `lib` folder with the following structure:

- components
  - index.js
  - index.d.ts
  - component
    - component.js
    - component.d.ts
- harmony
- hooks 
- and other...

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { defineConfig } from "vite";
+
+const files = fs.readdirSync("./src/harmony").filter((file) => !file.includes("index.ts"));
+
+const components = files.reduce((acc, component) => {
+    acc[component.split(".")[0]] = `src/harmony/${component}/${component}.tsx`;
+
+    return acc;
+}, {
+    index: 'src/harmony/index.ts'
```
Comment (LamaEats) [inline vite.config.js:11]: Maybe multiple enrtypoints for default lib components and harmomy?

**Review by tsumo — APPROVED** [review]

**Review by LamaEats — APPROVED** [review]

**Pull request merged**